src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/Review/index.tsx

```diff
@@ -76,14 +74,12 @@ const ContractInteractionReview = ({ onClose, onPrev, tx }: Props): React.ReactE
     const txValue = tx.value ? toTokenUnit(tx.value, nativeCoin.decimals) : '0'
     dispatch(
       createTransaction({
-        safeAddress,
-        to: txRecipient,
+        safeAddress: safeAddress as string,
```

Comment:
I'm not convinced about this assertion.

As safeAddress may be `undefined` we prevent dispatching the action if safeAddress is `undefined`.